tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/internal/types/Tensors.java

```diff
@@ -0,0 +1,158 @@
+package org.tensorflow.internal.types;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.StringJoiner;
+import org.tensorflow.Tensor;
+import org.tensorflow.ndarray.NdArray;
+import org.tensorflow.ndarray.Shape;
+
+/**
+ * Tensor helper methods.
+ */
+public final class Tensors {
+
+  /**
+   * Prevent construction.
+   */
+  private Tensors() {
+  }
+
+  /**
+   * Equivalent to {@link #toString(Tensor, Integer) toString(tensor, null)}.
+   *
+   * @param tensor a tensor
+   * @return the String representation of the tensor
+   */
+  public static String toString(Tensor tensor) {
+    return toString(tensor, null);
+  }
+
+  /**
+   * @param tensor   a tensor
+   * @param maxWidth the maximum width of the output in characters ({@code null} if unlimited). This
+   *                 limit may surpassed if the first or last element are too long.
+   * @return the String representation of the tensor
+   */
+  public static String toString(Tensor tensor, Integer maxWidth) {
+    if (!(tensor instanceof NdArray)) {
```

Comment:
This will error on inputs like RawTensor, which should be supported.  Maybe take a look at `RawTensor.asTypedTensor`.  You'd have to move packages, but it seems like it would be useful.  @karllessard are there better ways to convert a tensor to an ndarray?

Reply:
I think I've now added support for `RawTensor`. Please review.